new/LOC7368Tiberias.xml

```diff
@@ -0,0 +1,56 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7368Tiberias" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ጥብርያዶስ</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-02-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="lake" sameAs="wd:Q126982">
+                    <placeName xml:id="n1" xml:lang="en">Tiberias</placeName>
+                    <placeName corresp="'#n1" xml:lang="gez">ጥብርያዶስ</placeName>
+                    <country ref="wd:Q801"></country>
```

Comment:
https://github.com/BetaMasaheft/Places/pull/84#discussion_r1093139597

Reply:
Now I understand, what you meant. I copied from the guidelines, but you wanted me to copy from GitHub Website. Besides: Is Kinneret english or rather (Modern) Hebrew for Lake Tiberias?

Reply:
please have another look at the correction suggested here https://github.com/BetaMasaheft/Places/pull/84#discussion_r1093139597, and copy over, thank you
note that there are 5 elements placeName and different links for country and region, so the easiest is simply to copy paste the entire bit

Reply:
we want that if someone looks for a place one finds it, whichever name it goes under he or she uses. Kinneret is also used in English https://forecast.israelinfo.co.il/kineret